qa/rpc-tests/feature_fedpeg.py

```diff
@@ -47,42 +48,74 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
 
+    def add_options(self, parser):
+        parser.add_option("--parent_binpath", dest="parent_binpath", default="",
+                          help="Use a different binary for launching nodes")
+        parser.add_option("--parent_bitcoin", dest="parent_bitcoin", default=False, action="store_true",
+                          help="Parent nodes are Bitcoin")
+
     def setup_network(self, split=False):
+        if self.options.parent_bitcoin and self.options.parent_binpath == "":
+            raise Exception("Can't run with --parent_bitcoin without specifying --parent_binpath")
 
+        self.nodes = []
+        self.extra_args = []
         # Parent chain args
-        self.extra_args = [[
-            # '-printtoconsole',
-            '-validatepegin=0',
-            '-anyonecanspendaremine',
-            '-initialfreecoins=2100000000000000',
-        ]] * 2
-
-        self.nodes = start_nodes(2, self.options.tmpdir, self.extra_args[:2], chain='parent')
+        for n in range(2):
+            if self.options.parent_bitcoin:
+                self.parent_chain = 'regtest'
+                rpc_u, rpc_p = rpc_auth_pair(n)
+                self.extra_args.append([
+                    "-printtoconsole",
+                    "-port="+str(p2p_port(n)),
+                    "-rpcuser="+rpc_u,
+                    "-rpcpassword="+rpc_p,
+                    "-rpcport="+str(rpc_port(n)),
+                    "-addresstype=legacy", # To make sure bitcoind gives back p2pkh no matter version
+                ])
+            else:
+                self.parent_chain = 'parent'
+                self.extra_args.append([
+                    "-printtoconsole",
+                    '-validatepegin=0',
+                    '-anyonecanspendaremine',
+                    '-initialfreecoins=2100000000000000',
+                ])
+            self.binary = self.options.parent_binpath if self.options.parent_binpath != "" else None
+            self.nodes.append(start_node(n, self.options.tmpdir, self.extra_args[n], binary=self.binary, chain=self.parent_chain))
+
         connect_nodes_bi(self.nodes, 0, 1)
         self.parentgenesisblockhash = self.nodes[0].getblockhash(0)
         print('parentgenesisblockhash', self.parentgenesisblockhash)
-        parent_pegged_asset = self.nodes[0].getsidechaininfo()['pegged_asset']
+        if not self.options.parent_bitcoin:
+            parent_pegged_asset = self.nodes[0].getsidechaininfo()['pegged_asset']
 
         # Sidechain args
+        self.fedpeg_script = "512103dff4923d778550cc13ce0d887d737553b4b58f4e8e886507fc39f5e447b2186451ae"
         parent_chain_signblockscript = '51'
         for n in range(2):
             rpc_u, rpc_p = rpc_auth_pair(n)
-            self.extra_args.append([
-                # '-printtoconsole',
+            args = [
+                "-printtoconsole",
```

Comment:
nit: can we set this to 0?

Reply:
I added a commit to do this. If you want me to squash it with the others, let me know..

Reply:
You commented out, not put to 0. Bitcoin Core 0.17 defaults to `1`.